soc/infineon/cat1b/cyw20829/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2023 Cypress Semiconductor Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_sources(soc.c)
+zephyr_include_directories(.)
+
+# Expedition family defines
+zephyr_compile_definitions_ifdef(CONFIG_SOC_FAMILY_INFINEON_CAT1 CY_USING_HAL)
+zephyr_compile_definitions_ifdef(CONFIG_SOC_FAMILY_INFINEON_CAT1B COMPONENT_CAT1B)
+zephyr_compile_definitions(COMPONENT_CM33)
+
+# Use custom linker script
+set(SOC_LINKER_SCRIPT ${ZEPHYR_BASE}/soc/infineon/cat1b/cyw20829/linker.ld CACHE INTERNAL "")
```

Comment:
you are setting a custom linker script, so why the other changes in Zephyr related to linker scripts ?

Reply:
it just adds specific bootstrap sections, rest RAM/ROM is used Zephyr default: https://github.com/zephyrproject-rtos/zephyr/pull/70961/files#diff-b8ef5cf2af3dcc18025c6641731242b834cf7fdb021d94a16ab579dd7772d65aR180

see my comment above with details.

Reply:
thanks for the explanation.